docker/duoadmin3/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+fastapi = "*"
+uvicorn = "*"
+duo-client = "*"
+
+[requires]
+python_version = "3.8"
```

Comment:
In the Dockerfile, you stated python 3.9.
Can we use 3.10?

Reply:
I'm encouraged to set it to python3.10.
pipenv locks to a specific python version; it is the best practice.

Reply:
I need to check, the pr is not ready, I need to find the minimum I need to make it work, at the moment if I only add pydentic it doesn't work.
I added fastapi to the image and it did work, it's temporary for now so I'd be able to wok on duo for xsiam.
When everything will wirk with the integration I'll test the images